### PR TITLE
perfecto-upload 0.0.4

### PR DESCRIPTION
**Pull request by sreevatsa-ramya**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [ ] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [ ] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [ ] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [ ] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/perfecto-upload/0.0.4/step.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    description: |
+      This usually gets generated by the the "Gradle Runner" or "XCode Archive & Export" step
+    is_required: true
+    summary: The app file you want to upload to Perfecto, usually $BITRISE\_APK\_PATH
+      or $BITRISE\_IPA\_PATH
+    title: A Bitrise generated APK or IPA path
+  upload_path: $BITRISE_APK_PATH
+- opts:
+    description: |
+      The security token you use to log into your Perfecto account
```
Comment (koral--) [inline steps/perfecto-upload/0.0.4/step.yml:48]: should be sensitive

File: steps/perfecto-upload/0.0.4/step.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    is_required: true
+    summary: Your perfecto access key.
+    title: Perfecto access key
+  perfecto_accesskey: $PERFECTO_ACCESS_KEY
+- opts:
+    description: |
+      The perfect cloud name
```
Comment (koral--) [inline steps/perfecto-upload/0.0.4/step.yml:55]: typo (missing o)

File: steps/perfecto-upload/0.0.4/step.yml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      or $BITRISE\_IPA\_PATH
+    title: A Bitrise generated APK or IPA path
+  upload_path: $BITRISE_APK_PATH
+- opts:
+    description: |
+      The security token you use to log into your Perfecto account
+    is_required: true
+    summary: Your perfecto access key.
+    title: Perfecto access key
+  perfecto_accesskey: $PERFECTO_ACCESS_KEY
```
Comment (koral--) [inline steps/perfecto-upload/0.0.4/step.yml:52]: Default value shouldn't be here, see my comment in previous PR.

File: steps/perfecto-upload/0.0.4/step.yml (hunk 1 above)
Comment (sreevatsa-ramya) [reply steps/perfecto-upload/0.0.4/step.yml:48]: Added

File: steps/perfecto-upload/0.0.4/step.yml (hunk 2 above)
Comment (sreevatsa-ramya) [reply steps/perfecto-upload/0.0.4/step.yml:55]: changed

File: steps/perfecto-upload/0.0.4/step.yml (hunk 3 above)
Comment (sreevatsa-ramya) [reply steps/perfecto-upload/0.0.4/step.yml:52]: removed

**Pull request closed without merging**